clang/test/Driver/sycl-offload-wrapper-compression.cpp

```diff
@@ -0,0 +1,16 @@
+///
+/// Check if '--offload-compress' and '--offload-compression-level' CLI
+/// options are passed to the clang-offload-wrapper.
+///
+
+// REQUIRES: zstd
```

Comment:
As this is a driver specific test just checking the `-###` there should be no zstd requirement for it to run.

Reply:
Fixed in https://github.com/intel/llvm/pull/15124/commits/970ad3563efe05d7c65bf0b54050de63de645be1